### ISSUE TITLE
cleaning DB does not work with H2

### ISSUE THREAD

**Issue by trks1970** [database-rider/database-rider#461]
Here is my test configuration:

`@DBRider
@DBUnit(caseSensitiveTableNames = true, escapePattern = "\"?\"", mergeDataSets = true, allowEmptyFields = true)
@DataSet(value = ["datasets/base_api_test.yaml"], cleanBefore = true )`

when I execute the test, I get the error:

`2022-06-14 11:30:26,746 - [Test worker] WARN  c.g.d.r.c.d.DataSetExecutorImpl - Could not clean database before test.
java.lang.RuntimeException: Could not execute statements:Feature not supported: "META"; SQL statement:
DELETE FROM INFORMATION_SCHEMA."USERS" where 1=1 [50100-212]
	at com.github.database.rider.core.dataset.DataSetExecutorImpl.executeStatements(DataSetExecutorImpl.java:422)
...
`

DBRider version: 1.32.2
H2 version: 2.1.212

**Comment by skylvsme** [comment]
Try to remove caseSensitiveTableNames = true

**Comment by trks1970** [comment]
No, that does not help.

**Comment by rmpestano** [comment]
Hi @trks1970, do you have a sample project to share?
All rider-core tests are [using H2](https://github.com/database-rider/database-rider/blob/3d4ea11a14b41df35c4221ef6e881a6f06a201bc/rider-core/pom.xml#L78) so maybe it's a small thing on your project

**Comment by trks1970** [comment]
I will provide one.

**Comment by trks1970** [comment]
So, here is the test project:
https://github.com/trks1970/dbrider-test

When trying to export data, it fails with:
`ERROR c.g.d.r.c.exporter.DataSetExporter Could not export dataset.
org.dbunit.dataset.DataSetException: org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CONSTANTS" not found; SQL statement:
select "CONSTANT_CATALOG", "CONSTANT_SCHEMA", "CONSTANT_NAME", "VALUE_DEFINITION", "DATA_TYPE", "CHARACTER_MAXIMUM_LENGTH", "CHARACTER_OCTET_LENGTH", "CHARACTER_SET_CATALOG", "CHARACTER_SET_SCHEMA", "CHARACTER_SET_NAME", "COLLATION_CATALOG", "COLLATION_SCHEMA", "COLLATION_NAME", "NUMERIC_PRECISION", "NUMERIC_PRECISION_RADIX", "NUMERIC_SCALE", "DATETIME_PRECISION", "INTERVAL_TYPE", "INTERVAL_PRECISION", "MAXIMUM_CARDINALITY", "DTD_IDENTIFIER", "DECLARED_DATA_TYPE", "DECLARED_NUMERIC_PRECISION", "DECLARED_NUMERIC_SCALE", "GEOMETRY_TYPE", "GEOMETRY_SRID", "REMARKS" from "CONSTANTS" [42102-214]`

When running the repository test:
`WARN  c.g.d.r.c.d.DataSetExecutorImpl Could not clean database before test.
java.lang.RuntimeException: Could not execute statements:Feature not supported: "META"; SQL statement:
DELETE FROM INFORMATION_SCHEMA."INFORMATION_SCHEMA_CATALOG_NAME" where 1=1 [50100-214]`

**Comment by trks1970** [comment]
Any progress on this?

**Comment by rmpestano** [comment]
Hey @trks1970, I didn't have time to try it. Does anyone from @database-rider/rider-community can give it a try?

**Comment by rmpestano** [comment]
@trks1970 I gave it a quick try and I think you forgot to push the datasets:
```
Caused by: java.lang.RuntimeException: The dataset '/data.yaml' is neither a valid URL nor could be found under 'resources' or 'resources/datasets/' directory.
```

**Comment by trks1970** [comment]
Run the class DataGenerator in the test folder.

**Comment by trks1970** [comment]
I found a solution for DataGeneratorTest. I had to add schema="PUBLIC" to the DBUnit annotation.

**Comment by rmpestano** [comment]
Awesome! Can you push the changes to your branch? I'd like to include your project in [rider-examples](https://github.com/database-rider/database-rider/tree/master/rider-examples) (if you allow) as we don't have any kotlin sample.

**Comment by trks1970** [comment]
Pushed, Of course, use it as you wish.
Also, I was able to get the "could not clean database" exception go away, but adding

`
SYSTEM_SCHEMAS.put(RiderDataSource.DBType.H2, Collections.singleton("INFORMATION_SCHEMA"));
`
to DBUnitConfig, however I did not test this thoroughly.

**Comment by rmpestano** [comment]
The fix is available on rider `1.33.0`, thank you for the help @trks1970!
